.github/workflows/current-workflow.yml

```diff
@@ -50,6 +50,47 @@ jobs:
         run: docker rm flask_backend_container
         
       - run: echo "Docker-and-Python-Flask-Service job's status is ${{ job.status }}."
+
+  Publish-Flask-Image-to-Docker-Hub:
+    name: Publish Docker Python Flask Image to Docker Hub
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+      attestations: write
+      id-token: write
+    steps:
+      - run: echo "The job was automatically triggered by a ${{ github.event_name }} event."
+      - run: echo "Job is running on a ${{ runner.os }} server hosted by GitHub"
+      - run: echo "Branch name is ${{ github.ref }} and repository is ${{ github.repository }}."
+      - name: Check out repository code
+        uses: actions/checkout@v4
+      - run: echo "The repository ${{ github.repository }} has been cloned to runner."
+      - run: echo "Workflow is now ready to test code on runner."
+
+      - name: Install required dependencies
+        run: |
+          cd ./code/backend
+          pip install -r requirements.txt
+
+      - name: Build the Docker Image
+        working-directory: ./code/backend
+        run: docker build -t flask_backend .
+      
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+        
+      - name: Tag flask_backend container image
+        run: docker tag flask_backend rzheng2024/seprojects-cs673olf24team6:flask_backend
+      - name: Upload flask_backend container image to Docker Hub registry
+        run: docker push rzheng2024/seprojects-cs673olf24team6:flask_backend
+      - name: Check if flask_backend container image was successfully uploaded to Docker Hub
+        run: docker pull rzheng2024/seprojects-cs673olf24team6:flask_backend
```

Comment:
Yea this is definetly not the correct pipeline changes

Reply:
THis is the correct branch 
feature/publish-workflow-images-to-GitHub-Package-Registry